draft-ietf-quic-http.md

```diff
@@ -402,15 +432,28 @@ without EHB set, followed by a HEADERS frame with EHB set.
 ### PRIORITY {#frame-priority}
 
 The PRIORITY (type=0x02) frame specifies the sender-advised priority of a stream
-and is substantially different from {{!RFC7540}}. In order to support ordering,
-it MUST be sent only on the control stream. The format has been modified to
-accommodate not being sent on-stream and the larger stream ID space of QUIC.
+and is substantially different in format from {{!RFC7540}}.  In order to ensure
+that prioritization is processed in a consistent order, PRIORITY frames MUST be
+sent on the control stream.  A PRIORITY frame sent on any other stream MUST be
+treated as a HTTP_WRONG_STREAM error.
 
-The semantics of the Stream Dependency, Weight, and E flag are the same as in
-HTTP/2.
+The format has been modified to accommodate not being sent on a request stream,
+to allow for identification of server pushes, and the larger stream ID space of
+QUIC.  The semantics of the Stream Dependency, Weight, and E flag are otherwise
+the same as in HTTP/2.
 
 The flags defined are:
 
+  PUSH_PRIORITIZED (0x04):
+  : Indicates that the Prioritized Stream is a server push rather than a
+    request.  If set, this flag indicates that the Prioritized Stream identifies
+    a response stream and the Promise Index field is present.
+
+  PUSH_DEPENDENT (0x02):
+  : Indicates a dependency on a pushed request.  If set, this flag indicates
+    that the Dependent Stream identifies a response stream and that the
```

Comment:
Saying that it "identifies a response stream" in both these places feels a little odd.  If it's the stream ID, then just say it's the stream ID and you don't even need the flags.  If it's the Push ID, use the term here for clarity.